### DISCUSSION TITLE
Về algorithm environment

### DISCUSSION THREAD

**Discussion by TKhawns** [leduyquang753/uet-graduation-latex#5] [Q&A]
Mình đang sử dụng file .cls ở commit 10 và mình muốn thêm Pseudo code sử dụng `algorithm` và `algorithmic` environment. Hiện tại khi `usepackage` 2 môi trường trên thì mình gặp lỗi "undefined control sequence" ở `\begin{table}{caption}`. Làm thế nào để mình sử dụng được 2 package này ?

**Comment by leduyquang753** [comment]
(Chuyển sang issue #6.)
